src/setnpmreg.ps1

```diff
@@ -0,0 +1,8 @@
+If ((Test-Connection git.autodesk.com).PingSucceeded){
+    echo host reachable
+    npm set registry https://npm.autodesk.com/artifactory/api/npm/autodesk-npm-virtual/
+}
+else{
+    echo host not reachable
+    npm set registry https://registry.npmjs.org
+}
```

Comment:
`npm set registry` will override the global registry config.
Maybe we can add adsk and non-adsk .npmrc files. We could add these as part of a tools nuget package...or part of DynamoDS tools

Reply:
we discussed in standup to try copying npmrc files into the project directory before running npm commands - I will try that.